app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2><%= @task[:name] %></h2>
```

Comment:
You should use the dot methods to retrieve the values so rather than this "hash" syntax, it should be `@task.name`